### DISCUSSION TITLE
Books Not Importing and Scan Says Library Is Empty ***SOLVED***

### DISCUSSION THREAD

**Discussion by searay345** [advplyr/audiobookshelf#434] [Q&A]
Greetings!

So I just installed this on a Raspberry Pi 4 running OMV6 and Portainer. The audiobookshelf container works as expected. I can log into the WebUI just fine but I cannot see any books. I have 2 books in MP3 format in the Audiobooks folder. I double checked my Docker Compose path for Audiobooks and it is correct. Here's my structure:

services:
  audiobookshelf:
    image: advplyr/audiobookshelf
    ports:
      - 13378:80
    volumes:
      - /srv/dev-disk-by-uuid-7813e18c-cb11-43b4-9f5f-e7e28b068d00/Audiobooks:/audiobooks
      - /srv/dev-disk-by-uuid-7813e18c-cb11-43b4-9f5f-e7e28b068d00/Audiobooks/metadata:/metadata
      - /srv/dev-disk-by-uuid-7813e18c-cb11-43b4-9f5f-e7e28b068d00/Config/AudioBookShelf:/config
    restart: unless-stopped

I would really love to use this. What am I missing? Thanks for any help in advance.

**Comment by searay345** [comment]
Nevermind. I found the problem. I had the audiobook files in the root directory! Once I put the books in folders inside the root directly, it worked like a charm!

**Reply by advplyr** [reply]
Glad you got it working!
